docker-compose.yml

```diff
@@ -12,6 +12,20 @@ services:
       - mysql:/var/lib/mysql
     healthcheck:
       test: ["CMD", "mysqladmin", "ping", "--silent"]
+  nginx:
+    build:
+      context: .
+      dockerfile: nginx.Dockerfile
```

Comment:
You should use an image instead of building locally. You should update the actions to build this as well.

Reply:
Roger that!

Reply:
Addressed